### PR TITLE
SDK-173 update native sdks

### PR DESCRIPTION
**Pull request by echo-branch**
Updates iOS native SDK to 0.25.11.
Updates Android native SDK to 3.0.4.
Also update Android build update script and support the change from jar to aar.

### CODE REVIEW

File: src/ios/dependencies/Branch-SDK/Branch.m [hunk 1]
```diff
@@ -379,12 +396,13 @@ + (void) setBranchKey:(NSString*)branchKey {
                 @"You are using your test app's Branch Key. "
                  "Remember to change it to live Branch Key for production deployment."
             );
-        } else
-        if ([branchKey hasPrefix:@"key_live"]) {
+        
+        } else if ([branchKey hasPrefix:@"key_live"]) {
```
Comment (csalmi-branch) [inline src/ios/dependencies/Branch-SDK/Branch.m]: I noticed that there are a lot of style changes. Do we have a standard style guide or checker?

File: src/scripts/npm/updateNativeSdk.sh [hunk 2]
```diff
@@ -128,7 +116,7 @@ main() {
   fi
 
   # update plugin.xml
-  gulp prod
+  #gulp prod
```
Comment (csalmi-branch) [inline src/scripts/npm/updateNativeSdk.sh]: Do you mean to have this commented out?

File: src/scripts/npm/updateNativeSdk.sh [hunk 3]
```diff
@@ -62,23 +62,11 @@ update_ios() {
 }
 
 update_and() {
-  # properties
```
Comment (csalmi-branch) [inline src/scripts/npm/updateNativeSdk.sh]: Was this all removed due to the switch to using the aar?

**Comment by csalmi-branch** [comment]
Overall it looks good. Lets walk through it tomorrow and build it on my machine.

File: src/ios/dependencies/Branch-SDK/Branch.m (hunk 1 above)
Comment (echo-branch) [reply src/ios/dependencies/Branch-SDK/Branch.m]: For the most part, it's up to the developer.  Though we should be following Apple's guidelines whenever possible.

File: src/scripts/npm/updateNativeSdk.sh (hunk 2 above)
Comment (echo-branch) [reply src/scripts/npm/updateNativeSdk.sh]: Yes.  The gulpfile was removed with version 2.7.0, so this line errors out.  It appears it was used to update the xml file with all the iOS filenames.

File: src/ios/dependencies/Branch-SDK/Branch.m (hunk 1 above)
Comment (csalmi-branch) [reply src/ios/dependencies/Branch-SDK/Branch.m]: Can we enforce a style guide for consistency (at least for changes made by Branch employees)?

File: src/scripts/npm/updateNativeSdk.sh (hunk 3 above)
Comment (echo-branch) [reply src/scripts/npm/updateNativeSdk.sh]: Yes.

File: src/ios/dependencies/Branch-SDK/Branch.m (hunk 1 above)
Comment (echo-branch) [reply src/ios/dependencies/Branch-SDK/Branch.m]: I can look into it.  Haven't had the need before.

**Pull request merged**